commands/deploy.go

```diff
@@ -244,6 +244,13 @@ func addFileToTarWriter(filePath string, tarWriter *tar.Writer, prefix string) e
 	if err != nil {
 		return err
 	}
+	if stat.Mode()&os.ModeSymlink != 0 {
```

Comment:
Technically more correct where the mask is multibit and good habit so you can ignore how many bits the flag/mask is.
```suggestion
	if stat.Mode()&os.ModeSymlink == os.ModeSymlink {
```